restres/index.html

```diff
@@ -0,0 +1,151 @@
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-sm-9">
+                <form class="form">
+                    <div class="form-group">
+                        <label for="name3" >NAME </label> <input type="text" class="form-control" id="name3" placeholder="enter name" required>
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-sm-9">
+                <form class="form">
+                    <div class="form-group">
+                        <label for="phone" >PHONE</label> <input type="tel" class="form-control" id="phone" placeholder="enter phone" required>
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-sm-9">
+                <form class="form">
+                    <div class="form-group">
+                        <label for="email" >EMAIL</label> <input type="email" class="form-control" id="email" placeholder="abc@domain.com" required>
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-sm-4">
+                <form class="form">
+                    <div class="form-group">
+                        <label for="party" >PARTY SIZE</label> <input type="number" class="form-control" id="party" min="2" max="10">
+                    </div>
+                </form>
+            </div>
+
+            <div class="col-sm-5">
+                <form class="form">
+                    <div class="form-group">
+                        <label for="time1">OCCASION </label> <input type="text" class="form-control" id="time1" placeholder="Anniversary,bdays..">
+                    </div>
+                </form>
+            </div>
+        </div>
+        <button class="bg bg-danger btn-sm" > RESERVE</button>
+        <br>
+        <a href="edit.html" class="confirm" title="Click to go to user's options."> <em>Already have your confirmation number??</em> </a>
+    </div>
+</section>
+<section class="owner">
+
+    <label class="abc" for="email1" >OWNER-LOGIN &nbsp;</label>
+    <input class="login" type="email"  id="email1"/>
+
+    <label class="abc" for="password1" >PASSWORD &nbsp; &nbsp; &nbsp; &nbsp;</label>
+    <input type="password"  id="password1"/>
+    <button class="ow1 text-danger"> <a href="owner.html">LOGIN</a></button>
+</section>
+
+<section class="add">
+
+    <div> WELCOME ALL TO <strong> RITZ DINING! </strong> </div>
+    <div> TODAY SPECIAL OFFER !!!!!
+        10% DISCOUNT on TOTAL ORDER  :)
+        :)</div>
+    <input type="submit" id="ritz" value=" CONTINUE ">
+</section>
+
+<footer>
+    <p class="foot"> @2015 Proudly Designed and Developed by Ritanshi Parmar</p>
+
+    <!-- JavaScripts libraries-->
```

Comment:
load the js files at the end of the body (right before </body>) and not in the footer.